protocol/avdl/stellar1/local.avdl

```diff
@@ -519,6 +519,12 @@ protocol local {
   // airdropRegisterLocal registers a user for an airdrop.
   void airdropRegisterLocal(int sessionID, boolean register);
 
+  // Trustline operations:
+  // In addTrustline, limit is optional, if not supplied, max limit will be used.
+  void addTrustlineLocal(int sessionID, AccountID accountID, Trustline trustline, string limit);
+  void deleteTrustlineLocal(int sessionID, AccountID accountID, Trustline trustline);
+  void changeTrustlineLimitLocal(int sessionID, AccountID accountID, Trustline trustline, string limit);
```

Comment:
I imagine frontend is going to need `listTrustlines`.  I know it is in the balances reply, but to keep it simple for their logic, let's add `listTrustlines` that just returns the non-XLM trustlines for an account.

Reply:
ok this shouldn't be too hard to do - thanks!